spirv_msl.hpp

```diff
@@ -92,6 +93,8 @@ class CompilerMSL : public CompilerGLSL
 	// Compiles the SPIR-V code into Metal Shading Language using default configuration parameters.
 	std::string compile() override;
 
+	void set_entry_point_name(std::string func_name);
+
```

Comment:
For consistency, can we pass an alternate main function name in `MSLConfiguration` instead?